Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/MaxLedInit.c

```diff
@@ -0,0 +1,161 @@
+/** @file
+
+  Copyright (c) 2020 - 2023, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <PiPei.h>
+#include <Library/BaseLib.h>
+#include <Library/DebugLib.h>
+#include <Library/IoLib.h>
+#include <Library/PchPcrLib.h>
+#include <Library/PciCf8Lib.h>
+#include <Library/PciLib.h>
+#include <Library/PchInfoLib.h>
+#include <Library/TimerLib.h>
+#include <PchAccess.h>
+#include <Register/PchRegsLpc.h>
+#include <Register/PchRegsPcr.h>
+#include <Library/PchPciBdfLib.h>
+#include <PlatformBoardId.h>
+#include <Library/SmbusLib.h>
+#include <Library/SerialIoI2cLib.h>
```

Comment:
It looks some header files are not really used. better only include necessary files.

Reply:
removed header files that are not needed